code-scanning/properties/clj-holmes.properties.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "clj-holmes",
+    "creator": "Matheus Bernardes",
+    "description": "A CLI SAST (Static application security testing) tool which was built with the intent of finding vulnerable Clojure code via rules that use a simple pattern language.",
```

Comment:
```suggestion
    "description": "A Static Application Security Testing tool to find vulnerable Clojure code via rules that use a simple pattern language.",
```
The setup tiles generally look better with a slightly shorter description, and in this context users won't be employing the tool as a CLI directly. 🙂